Base/Logic/Resources/ColorFiles/Terminology/GenericAnatomyColors-SNOMED.csv

```diff
@@ -0,0 +1,291 @@
+# Integer Label,Text Label,Segmented Property Category -- CID 7150++,Segmented Property Type,Segmented Property Type Modifier,Color
+# Version 2
+SlicerLUT=GenericAnatomyColors
+1,tissue,(T-D0050;SRT;Tissue),(T-D0050;SRT;Tissue),,rgb(128;174;128)
```

Comment:
Need to add to the commit message that this commit adds a number of SNOMED-CT (https://www.nlm.nih.gov/research/umls/Snomed/snomed_main.html) codes to Slicer source code repository. All of those codes we are adding are also included in the DICOM standard, and therefore are subject to the SNOMED-DICOM reuse agreement [1].

[1] ftp://medical.nema.org/medical/dicom/Geninfo/Policies/code_policy_20000921.pdf